sonic-xcvrd/xcvrd/sff_mgr.py

```diff
@@ -435,6 +436,17 @@ def task_worker(self):
                     # Skip if these essential routines are not available
                     continue
 
+                set_lp_success = (
```

Comment:
I assume lpmode handling is only needed at module insertion event, right?

Just some minor suggestion:
Maybe adding below condition check can avoid unnecessary lpmode handling in other cases (e.g.   the case of admin_status/host_tx_ready getting changed by `config interface shutdown/startup`)
```
if xcvr_inserted:
    <lpmode logic>
```

Reply:
The described scenario unfortunately still requires lpmode set False. If you shutdown then startup the interface without bringing it out of lpmode the interface remains down even if it was up before shutdown

Reply:
My mistake, I believe you are correct. I'll add that change, thanks